src/app/components/RedButton.js

```diff
@@ -0,0 +1,11 @@
+export const RedButton = (text = '') => {
+  const redButton = document.createElement('button');
+  redButton.classList.add('redButton');
+  redButton.textContent = text;
+
+  redButton.addEventListener('click', () => {
```

Comment:
O to chodzi, żeby nasz button reagował na click, ale przydałoby się, żeby funkcja którą odpala była parametrem buttona :) Wtedy możemy go requżyć i tylko przekazac funkcję, którą wykona na onClick.

Reply:
Chociaż jeszcze też coś nie działa  w tym rozwiązaniu :/ Bo jak odpaliłem konsolę to nie pokazuje się ten log przy klikaniu. Test z renderowaniem powinien to wykryć :)

Reply:
Dziwne. U mnie console.log działa 👍 

![image](https://user-images.githubusercontent.com/57037612/103458617-34749780-4d0a-11eb-963c-30a2f1c7b43f.png)

Reply:
Tak jak Mateusz napisał - callback function jest potrzebna

Reply:
Racja działa - miałem filtrowanie konsoli :)